namespaces/live-1.cloud-platform.service.justice.gov.uk/court-probation-prod/resources/crime-portal-mirror-gateway-route53.tf

```diff
@@ -0,0 +1,23 @@
+resource "aws_route53_zone" "crime_portal_mirror_gateway_route53_zone" {
+  name = var.crime-portal-mirror-gateway-domain
+
+  tags = {
+    business-unit          = var.business-unit
+    application            = var.application
+    is-production          = var.is-production
+    environment-name       = var.environment-name
+    owner                  = var.team_name
+    infrastructure-support = var.infrastructure-support
+  }
+}
+
+resource "kubernetes_secret" "crime_portal_mirror_gateway_route53_zone_sec" {
+  metadata {
+    name      = "crime-portal-mirror-gateway-route53-zone-output"
+    namespace = var.namespace
+  }
+
+  data = {
+    zone_id = aws_route53_zone.route53_zone.zone_id
```

Comment:
`aws_route53_zone.route53_zone.zone_id` should be `aws_route53_zone.crime_portal_mirror_gateway_route53_zone.zone_id` here

Reply:
Thanks for the response, this has now been changed